src/@sdk/mutations/checkout.ts

```diff
@@ -170,14 +169,10 @@ export const removeCheckoutPromoCode = gql`
       checkout {
         ...Checkout
       }
-      errors {
-        field
-        message
-      }
       checkoutErrors {
+        code
         field
         message
-        code
```

Comment:
If we would extract this to fragment, it would save us some lines of code and simplify typing in the components accepting these errors

Reply:
yep, right